stories/AlertBar.stories.js

```diff
@@ -22,27 +23,35 @@ storiesOf('AlertBar', module)
     .add('Default', () => <AlertBar>Default - I will autohide</AlertBar>)
     .add('States', () => (
         <React.Fragment>
-            <AlertBar permanent>Default (info)</AlertBar>
-            <AlertBar permanent success>
+            <AlertBar className="first" permanent>
```

Comment:
I vaguely recollect that it would be preferred to work with `data-test-id` attributes for selecting elements in functional tests. Not sure why that would be better tbh...

Did we decide that classnames are just as good?

Reply:
I think we've decided on `data-test-id`, but we haven't decided on how to add them yet (although manually adding them seems the only do-able option to me).
But as there was no api for this yet on this component (or any other in `ui-core`), I think using the className is just as fine for now.

I'm not sure we should add that api either, in the import export app I've wrapped the ui-core components with an element that has the id, so it's purely the apps responsibility to take care of these IDs